go.mod

```diff
@@ -0,0 +1,54 @@
+module github.com/stolostron/rbac-api-utils
+
+go 1.19
+
+require (
+	golang.org/x/exp v0.0.0-20230108222341-4b8118a2686a
+	k8s.io/api v0.26.0
+	k8s.io/apimachinery v0.26.0
+	k8s.io/client-go v0.26.0
```

Comment:
You may want to use lower versions since Observability may not be using v0.26.0 yet and this would force them to upgrade.

One reason not to upgrade to newer versions of these Kubernetes modules is that it removes the option for `configmap` leader election which is required for OCP 3.11. This is not applicable for Observability since it's on the Hub, but there may be other reasons.

Reply:
Obs is at v0.25.2 .. downgraded this to v0.25.2 as well